test/test_binary_ufuncs.py

```diff
@@ -738,6 +738,30 @@ def test_complex_scalar_pow_tensor(self, device, dtype):
         for base in complexes:
             self._test_pow(base, exp)
 
+    @onlyOnCPUAndCUDA
+    def test_pow_scalar_type_promotion(self, device):
+        # We expect the computation to be performed in uint8 (overflowing to 0), and then cast to int64
+        input_tensor_uint8 = torch.tensor([17], dtype=torch.uint8, device=device)
```

Comment:
it will be slightly more readable and better show the purpose of the test if you create a tuple of (scalar, size=1) inputs, and then have a loop `for input in inputs` instead of repeating test block.